docs/deployment/feature_gates.md

```diff
@@ -108,3 +111,6 @@ A *General Availability* (GA) feature is also referred to as a *stable* feature.
 * `DenyInvalidExtensionResources` causes the `seed-admission-controller` to deny invalid extension resources, instead of just logging validation errors.
 * `WorkerPoolKubernetesVersion` allows to overwrite the Kubernetes version used for shoot clusters per worker pool (see [this document](../usage/worker_pool_k8s_versions.md))
 * `CopyEtcdBackupsDuringControlPlaneMigration` enables the copy of etcd backups from the object store of the source seed to the object store of the destination seed during control plane migration.
+* `SecretBindingProviderPopulator` enables the SecretBinding provider populator controller in the gardener-controller-manager. The controller is responsible to populate the provider.type of the SecretBinding resource based on its current usage.
```

Comment:
Nit: How about renaming this controller to simply `SecretBindingProvider`? There are no other secret binding provider controllers (and it's unlikely to have them soon), we could still keep the documentation and other logic related to "populating" but use a shorter name.